common/src/main/java/com/wynntils/models/lootrun/scoreboard/LootrunScoreboardPart.java

```diff
@@ -56,6 +58,11 @@ public void onSegmentChange(ScoreboardSegment newValue) {
         } else if (currentStateLine.matches(DEFEND_PATTERN, PartStyle.StyleType.NONE)) {
             Models.Lootrun.setState(LootrunningState.IN_TASK, LootrunTaskType.DEFEND);
         }
+
+        currentStateLine = content.get(1);
```

Comment:
I also had a few errors pop up from this whilst I was testing the regex by doing /class to pause the timer and come back, so you probably want a check that content does have an index 1